marketplace/deployer_envsubst_base/create_manifests.sh

```diff
@@ -45,6 +45,8 @@ mkdir "$manifest_dir"
 if [[ "$mode" = "test" ]]; then
   if [[ -e "/data-test" ]]; then
     cp -RT "/data-test" "/data"
+  else
+    echo "$LOG_SMOKE_TEST ERROR Missing /data-test."
```

Comment:
Is this an error? We'll simply skip if there is no `/data-test`, right?

Reply:
It is an error. But the test pipeline should keep going to collect more results to show in the report.